src/types/core.d.ts

```diff
@@ -2,7 +2,8 @@ export interface Identifiable {
   id: string
 }
 
-export interface Resource<R> {
+export interface Resource<R, E> {
+  included?: E
```

Comment:
I don't think you need this addition, I don't see you use it anywhere. If you do need somewhere then create a new interface rather than changing Resource, because it is used across the project.

Reply:
Thanks!